content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Shutting Down and Restart Cluster Gracefully"
+description: "Demonstrate how to shut down and restart kubernetes cluster Gracefully"
+layout: "single"
+
+linkTitle: "Shutting Down and Restart Cluster Gracefully"
+weight: 2000
+
+icon: "/images/docs/docs.svg"
+---
+This document describes the process to gracefully shut down your cluster. You might need to temporarily shut down your cluster for maintenance reasons.
+
+{{< notice warning >}}
+Shutting down cluster is very dangerous, you must to understand what are you doing and make an etcd backup before proceeding.
+Usually we recommend one by one maintenance nodes instead of restarting cluster.
```

Comment:
Usually, it is recommended to maintain your nodes one by one instead of restarting the whole cluster.